ERROR:__main__:Exception Datetime: 15-03-2024 10:25:30 AM
ERROR:__main__:Exception Type: <class 'exceptions.InvalidDataFileError'>
ERROR:__main__:Exception Value: 
ERROR:__main__:Message Value: The specified data file was not found or does not contain valid data.
ERROR:__main__:File Name: C:\Users\eha\PycharmProjects\Hausarbeit\datasets.py
ERROR:__main__:Line Number: 75
ERROR:__main__:Procedure Name: __init__
ERROR:__main__:Line Code: raise InvalidDataFileError
ERROR:__main__:Exception Datetime: 15-03-2024 10:26:41 AM
ERROR:__main__:Exception Type: <class 'FileNotFoundError'>
ERROR:__main__:Exception Value: [Errno 2] No such file or directory: 'thisfiledoesnotexist.csv'
ERROR:__main__:File Name: C:\Users\eha\.conda\envs\Hausarbeit\lib\site-packages\pandas\io\common.py
ERROR:__main__:Line Number: 859
ERROR:__main__:Procedure Name: get_handle
ERROR:__main__:Line Code: handle = open(
ERROR:__main__:The following file was not found: thisfiledoesnotexist.csv
ERROR:__main__:Exception Datetime: 15-03-2024 10:27:35 AM
ERROR:__main__:Exception Type: <class 'FileNotFoundError'>
ERROR:__main__:Exception Value: [Errno 2] No such file or directory: 'thisfiledoesnotexist.csv'
ERROR:__main__:File Name: C:\Users\eha\.conda\envs\Hausarbeit\lib\site-packages\pandas\io\common.py
ERROR:__main__:Line Number: 859
ERROR:__main__:Procedure Name: get_handle
ERROR:__main__:Line Code: handle = open(
ERROR:__main__:The following file was not found: thisfiledoesnotexist.csv
ERROR:__main__:Exception Datetime: 15-03-2024 10:27:44 AM
ERROR:__main__:Exception Type: <class 'FileNotFoundError'>
ERROR:__main__:Exception Value: [Errno 2] No such file or directory: 'thisfiledoesnotexist.csv'
ERROR:__main__:File Name: C:\Users\eha\.conda\envs\Hausarbeit\lib\site-packages\pandas\io\common.py
ERROR:__main__:Line Number: 859
ERROR:__main__:Procedure Name: get_handle
ERROR:__main__:Line Code: handle = open(
ERROR:__main__:The following file was not found: thisfiledoesnotexist.csv
ERROR:__main__:Exception Datetime: 15-03-2024 10:28:20 AM
ERROR:__main__:Exception Type: <class 'FileNotFoundError'>
ERROR:__main__:Exception Value: [Errno 2] No such file or directory: 'thisfiledoesnotexist.csv'
ERROR:__main__:File Name: C:\Users\eha\.conda\envs\Hausarbeit\lib\site-packages\pandas\io\common.py
ERROR:__main__:Line Number: 859
ERROR:__main__:Procedure Name: get_handle
ERROR:__main__:Line Code: handle = open(
ERROR:__main__:The following file was not found: thisfiledoesnotexist.csv
ERROR:__main__:Exception Datetime: 15-03-2024 10:28:58 AM
ERROR:__main__:Exception Type: <class 'FileNotFoundError'>
ERROR:__main__:Exception Value: [Errno 2] No such file or directory: 'thisfiledoesnotexist.csv'
ERROR:__main__:File Name: C:\Users\eha\.conda\envs\Hausarbeit\lib\site-packages\pandas\io\common.py
ERROR:__main__:Line Number: 859
ERROR:__main__:Procedure Name: get_handle
ERROR:__main__:Line Code: handle = open(
ERROR:__main__:The following file was not found: thisfiledoesnotexist.csv
